package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@dated/transaction-export-plugin",
-  "version": "1.1.3",
+  "version": "1.1.4",
```

```suggestion
  "version": "1.1.3",
```
i'll bump the version when doing the release.